python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -883,6 +931,8 @@ def startup(self):
             self.repository.startup()
             # All Ids could have changed
             self.changed_ids = {}
+            self.flush_thread = RegistryFlusher(self)
+            self.flush_thread.start()
```

Comment:
Every subclass calls (or doesn't override) this method and so every one will have a flusher. About per-registry configuration, I'm not sure. What are the reasons for and against?